src/pytesseract.py

```diff
@@ -98,63 +102,54 @@ def prepare(image):
 
     raise TypeError('Unsupported image object')
 
+def run_and_get_output(image, extension, lang=None, config=None, nice=None):
+    temp_name = ''
+    try:
+        temp_name = save_image(image)
+        input_filename = temp_name+'.bmp'
+        output_filename_base = temp_name+'_out'
+        run_tesseract(input_filename, output_filename_base, extension, lang, config, nice)
+        with open(output_filename_base+'.'+extension, 'rb') as output_file:
+            return output_file.read().decode('utf-8').strip()
+    finally:
+        cleanup(temp_name)
 
-def image_to_string(image, lang=None, boxes=False, config=None, nice=0, verbose=0):
+def image_to_string(image, lang=None, config=None, nice=0):
```

Comment:
One small nitpick - since the image_to_string function signature is changed, some end users might get TypeError for unexpected argument, because the boxes flag is not available anymore.

Maybe we can add deprecation warning that suggests using the newly available function and aborting the function gracefully, so the users can migrate more easily, instead of just getting confusing exceptions at first.

I can include this if you want.

Reply:
I actually had this change locally and forgot to commit it. My thoughts exactly :)